common/app/common/dfp/TagSponsorship.scala

```diff
@@ -77,13 +92,38 @@ object HighMerchandisingTargetedTagSet {
   }
 }
 
+
 case class HighMerchandisingTargetedTagSet(items: Set[String] = Set.empty){
 
   def hasTag (tag: Tag): Boolean = items.exists(item => tag.id.endsWith(item))
 
+  def hasAdUnit (adUnitSuffix : String): Boolean = {
+    println(adUnitSuffix)
+    true
+  }
+
 
   def nonEmpty = items.nonEmpty
+}
+
+case class HighMerchandisingLineItemTargetsSeq(lineItems : Seq[HighMerchandisingLineItem] = Seq.empty) {
```

Comment:
@kelvin-chappell @rich-nguyen have now included all in HighMerchandisingLineItem case class rather than have a separate target case class.